maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/FdcContributionsService.java

```diff
@@ -145,4 +155,74 @@ private boolean saveErrorMessage(LogFdcProcessedRequest request, FdcContribution
         return debtCollectionService.saveError(ContributionFileUtil.buildContributionFileError(request, fdcEntity));
     }
 
+    public Integer createFdcItems(CreateFdcItemRequest fdcRequest) {
+
+        try {
+            log.info("Create createFdcItems {}", fdcRequest);
+
+            FdcItemsEntity fdcItemsEntity = FdcItemsEntity.builder()
+                .fdcId(fdcRequest.getFdcId())
+                .latestCostInd(fdcRequest.getLatestCostInd())
+                .itemType(fdcRequest.getItemType())
+                .adjustmentReason(fdcRequest.getAdjustmentReason())
+                .userCreated(fdcRequest.getUserCreated())
+                .paidAsClaimed(fdcRequest.getPaidAsClaimed())
+                .dateCreated(fdcRequest.getDateCreated().toLocalDate())
+                .build();
+
+            FdcItemsEntity savedEntity = fdcItemsRepository.save(fdcItemsEntity);
+            return savedEntity.getId();
+
+        } catch(Exception e){
```

Comment:
I'm not sure if we need these try catches, as if any exception is thrown the exception handler will manage it and respond with a request to the caller? And the log line above e.g.` log.info("Create createFdcItems {}", fdcRequest);`
should make it easier to debug if needed?

```
try {
...
} catch(Exception e){
```